spine/binding_manager_test.go

```diff
@@ -183,54 +183,104 @@ func (suite *BindingManagerSuite) Test_Bindings() {
 			Feature: util.Ptr(model.AddressFeatureType(1000)),
 		}),
 	}
-	err = bindingMgr.RemoveBinding(bindingDelete, suite.remoteDevice)
-	assert.NotNil(suite.T(), err)
+	err = bindingMgr.RemoveBinding(s.remoteDevice, bindingDelete)
+	assert.Nil(s.T(), err)
```

Comment:
Assuming we're purposefully considering deleting nonexistant bindings a success now, maybe we should add a comment like the following here to clarify.
```suggestion
        // removing a binding that doesn't exist is considered a success
	err = bindingMgr.RemoveBinding(s.remoteDevice, bindingDelete)
	assert.Nil(s.T(), err)
```